components/vf-badge/vf-badge.angular.component.ts

```diff
@@ -0,0 +1,40 @@
+/*This is the  code which needs to be built as Angular library and published as npm module before it can be used in any Angular project */
+import { Component, Input, OnInit } from "@angular/core";
+
+@Component({
+  selector: "vf-badge",
+  template: `
+    <a [attr.id] = "id !== undefined ? id : null" *ngIf="badge_href !== ''" [href]="badge_href" [class]="class">{{content}}</a>
+    <span [attr.id] = "id !== undefined ? id : null" *ngIf="!badge_href || badge_href === null || badge_href === ''" [class]="class">{{content}}</span>
+  `,
+  styleUrls: []
+})
+
+export class VfBadgeAngularComponent implements OnInit {
+  /* Initialize values based on input values */
+  @Input() badge_href = '';
+  @Input() theme = '';
+  @Input() text = '';
+  @Input() style = [];
```

Comment:
You could specify the different types of badges, like this: 
`@Input() style: Array<'tertiary' | 'secondary' | 'primary'> = [];`
It would be something nice to have, as a developer

Reply:
Thanks @pacope92. Changes added with latest commit. Please review.